tribuo-classification/README.md

```diff
@@ -0,0 +1,105 @@
+or begnin given certain input attributes.
+
+This is a simple app using the Conclave API. It is licensed under the Apache 2 license, and therefore you may 
+copy/paste it to act as the basis of your own commercial or open source apps.
+
+Use below link to start download Conclave and start building Conclave apps:
+
+https://conclave.net/get-conclave/
+
+##  Dataset sample used for this application.
+The dataset used for this sample has been taken from the below url. 
+https://archive.ics.uci.edu/ml/datasets/breast+cancer+wisconsin+(original)
+There are 11 columns in the data set. The first 10 columns reflect different attributes which help in detecting 
+if the breast cancer is malignant or benign. The last column is the class 2 for benign, 4 for malignant.
+
+##  Attribute                     Domain
+   -- -----------------------------------------
+1. Sample code number            id number
+2. Clump Thickness               1 - 10
+3. Uniformity of Cell Size       1 - 10
+4. Uniformity of Cell Shape      1 - 10
+5. Marginal Adhesion             1 - 10
+6. Single Epithelial Cell Size   1 - 10
+7. Bare Nuclei                   1 - 10
+8. Bland Chromatin               1 - 10
+9. Normal Nucleoli               1 - 10
+10. Mitoses                       1 - 10
+11. Class:                        (2 for benign, 4 for malignant)
+
+## Parties involved in the sample
+Hospitals, Host, Enclave.
+
+Hospitals - Hospitals are responsible for providing input data to train the model.
+
+Host - Hosts loads the enclave.
+
+Enclave - Enclave collates all the data given by all hospitals and trains the model. This also evaluates the model and
+sends the evaluation result back to the clients.
+
+## Evaluation result
+Below evaluation result gets printed out to all the clients
+
+| Class        | n           | tp  |fn    |fp    | recall           | prec  |f1 |
+| ------------- |-------------  | -----|------------- |:-------------:| -----:|-----:|-----:|
+| 2      | 264    |     0       |   264        |    0        | 0.000       |0.000          |   0.000 |
+|4                    |          156 |        156   |        0     |    264    |   1.000   |    0.371 |      0.542|
+Total                   |      420       |  156       |  264       |  264|
+Accuracy         | | | | |                                                          0.371
+Micro Average            | | | | |                                                     0.371   |    0.371   |    0.371|
+Macro Average                        | | | | |                                         0.500   |     0.186    |    0.271| 
+Balanced Error Rate        | | | | |                                                   0.500
+
+## How to run on a non-linux based system
+
+Start the host on a non-Linux system, which will build the enclave and host by default in simulation mode.
+
+    ./container-gradle host:run
```

Comment:
if you delete container gradle per rui's recommendation, be sure to update this to something like `<PATH-TO-CONCLAVE-INSTALL-DIRECTORY>/scripts/container-gradle`

Reply:
done